src/App.js

```diff
@@ -1,4 +1,4 @@
-import "./App.css";
+// import "./App.css";
 import {Routes, Route} from "react-router-dom";
 import {Home, LikedVideos, WatchLater, Login, Signup, VideoListing} from "../src/frontend/pages/index";
```

Comment:
appreciation: good manner of importing files.

Reply:
thanks